### PR TITLE
fixed end of line \ fold highlighting of next word

### PR DESCRIPTION
**Pull request by gabeio**
this fixes:

``` LiveScript
var something, something, something,\
    somethingelse, something
```

`somethingelse` will be highlighted wrong as the `\` will initiate the `\word` even if the end of line occurs

### CODE REVIEW

**Comment by blvz** [comment]
Thanks and sorry for the delay.

**Comment by gabeio** [comment]
it's totally okay! :+1: still by far the nicest ls highlighter around :smiley:

---

there were a few others (I can't remember off the top of my head) I was trying to fix but WOW that regex can be hard to follow :laughing:...

**Comment by blvz** [comment]
Hahaha. Yes. Those regexes are insane. But don't credit me on that, I've just ported from @farzher's tmbundle fork.

Check this: https://github.com/paulmillr/LiveScript.tmbundle/pull/39
